tensorflow/python/ops/clip_ops.py

```diff
@@ -48,6 +48,17 @@ def clip_by_value(t, clip_value_min, clip_value_max,
   Note: `clip_value_min` needs to be smaller or equal to `clip_value_max` for
   correct results.
 
+  For example:
+  
+  ```python
+  A=tf.constant([[1,20,13],[3,21,13]])
```

Comment:
Can the code be updated to be compatible with TF style? This means space around = used for assignment, space after commas, etc